zulipterminal/ui_tools/buttons.py

```diff
@@ -297,11 +297,18 @@ def __init__(
             stream_name=self.stream_name,
             topic_name=self.topic_name,
         )
+
+        topic_prefix = " "
+        topic_name = self.topic_name
+        if self.topic_name.startswith(CHECK_MARK + " "):
```

Comment:
This is defined more by the API than our internal symbols, so I'd suggest we put the constant there (eg. `RESOLVED_PREFIX`)